src/utils/dom.ts

```diff
@@ -274,3 +274,7 @@ export function intersects(rect1: DOMRect, rect2: DOMRect): boolean {
     rect2.bottom < rect1.top
   );
 }
+
+export function toAriaBoolean(value: boolean): string {
```

Comment:
Since this function could apply to non-ARIA attributes, how about `toBooleanAttr`?

Reply:
I thought about that but it seems like `aria-` is the **only** place we are using it and where it really should be used.

Reply:
Most boolean attributes we don't want to be strings. If we do need to rename, I would recommend `toBooleanString()` instead but I actually prefer `toAriaBoolean` because it says what it is for.

Reply:
Sold! `toAriaBoolean` it is.